src/scripts/public/sites/cleanbc/patterns/actions-toggle-query.js

```diff
@@ -162,7 +162,7 @@ const domReady = () => {
 
 	function countActiveProjects() {
 		const elAccordionItems = document.querySelectorAll(
-			'.wp-block-bcgov-collapse-item'
+			'.wp-block-bcgov-block-theme-collapse-item'
```

Comment:
is this code really just used for cleanbc patterns, and not reusable anywhere else, this seems like global work, not cleanbc specific.

Reply:
This is legacy structure and needs to be refactored to live outside the CleanBC directory.